src/app/modules/main/communities/minting/io_interface.nim

```diff
@@ -7,9 +9,9 @@ method delete*(self: AccessInterface) {.base.} =
 method load*(self: AccessInterface) {.base.} =
   raise newException(ValueError, "No implementation available")
 
-method mintCollectible*(self: AccessInterface, address: string, name: string, symbol: string, description: string, supply: int, infiniteSupply: bool, transferable: bool,
+method mintCollectible*(self: AccessInterface, communityId: string, address: string, name: string, symbol: string, description: string, supply: int, infiniteSupply: bool, transferable: bool,
                       selfDestruct: bool, network: string) {.base.} =
```

Comment:
We need to update that type since the needed information is the `chainId`:

```suggestion
                      selfDestruct: bool, network: int) {.base.} =
```